extensions/mssql/src/objectExplorerNodeProvider/objectExplorerNodeProvider.ts

```diff
@@ -100,15 +77,15 @@ export class MssqlObjectExplorerNodeProvider extends ProviderBase implements sql
 	}
 
 	private async doExpandNode(nodeInfo: sqlops.ExpandNodeInfo, isRefresh: boolean = false): Promise<boolean> {
-		let session = this.sessionMap.get(nodeInfo.sessionId);
+		let sqlClusterSession = this.sessionMap.get(nodeInfo.sessionId);
```

Comment:
Again, please remove unnecessary naming changes so we can shorten the reviews.

Reply:
rolled back.